WorldProcessor/WorldProcessor.Core/Entities/Tile.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace WorldProcessor.Core.Entities
+{
+    public class Tile : ICloneable
+    {
+        public string ConfigId { get; set; }
```

Comment:
It is not a good practice, yes. But it happened because objects mutates in iteration and generation algorithms. I suggest to fix it later while WorldProcessor enhancement.